### ISSUE TITLE
Rule proposal: disallow no extension imports (folders, file without extension)

### ISSUE THREAD

**Issue by vvo** [benmosher/eslint-plugin-import#204]
I would like to enforce this:

```js
import './file.js';
import './dir/index.js';
```

versus

```js
import './file';
import './dir/';
```

While still being able to require files that have no extension of course.

What do you think?

Is that doable?

**Comment by benmosher** [comment]
Makes sense. Should be pretty straightforward. 

Could be an extension to `no-unresolved`, or a separate rule. The former should be more efficient.

**Comment by vvo** [comment]
Nice!! Thanks community :D
